.github/workflows/main.yaml

```diff
@@ -8,7 +8,7 @@ on:
 
 
 env:
-  WEAVIATE_VERSION: preview--f005cb8
+  WEAVIATE_VERSION: 1.24.1
```

Comment:
you can switch to 1.24.12

Reply:
Ye gonna update this to `1.25.0` in a separate PR. I want the changes to the test to be `main` more than the CI